tests/unit/h/accounts/schemas_test.py

```diff
@@ -156,9 +155,7 @@ def test_it_validates_with_valid_payload(
 
         result = schema.deserialize(valid_params)
 
-        assert result == dict(
-            valid_params, privacy_accepted=True, comms_opt_in=None, csrf_token=None
-        )
```

Comment:
The test was expecting the Colander schema to include `csrf_token: None` in the deserialized data if there was no `csrf_token` in the input. I've changed it to omit `csrf_token` from the output in this case.